### PR TITLE
add nextcord.Spotify.elapsed

### LINKED ISSUE #117: add discord.Spotify.elapsed
### Link

https://github.com/Rapptz/discord.py/pull/7069

### PR DESCRIPTION
**Pull request by ooliver1**
## Summary

closes #117
uses tz aware time to calculate the elapsed time since a current spotify activity song started

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

++I cant tell if theres any autoformatting here, it doesnt show in editor but shows sometimes on github

### CODE REVIEW

File: nextcord/activity.py [hunk 1]
```diff
@@ -223,7 +223,8 @@ def __init__(self, **kwargs):
         self.timestamps: ActivityTimestamps = kwargs.pop('timestamps', {})
         self.assets: ActivityAssets = kwargs.pop('assets', {})
         self.party: ActivityParty = kwargs.pop('party', {})
-        self.application_id: Optional[int] = _get_as_snowflake(kwargs, 'application_id')
+        self.application_id: Optional[int] = _get_as_snowflake(
```
Comment (TAG-Epic) [inline nextcord/activity.py:226]: Autoformatting was turned on, please revert your autoformatter's changes

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: nextcord/activity.py [hunk 2]
```diff
@@ -706,6 +711,14 @@ def party_id(self) -> str:
         """:class:`str`: The party ID of the listening party."""
         return self._party.get('id', '')
 
+    @property
+    def elapsed(self) -> datetime.timedelta:
+        """:class:`datetime.timedelta`: Time elapsed since this song.
+
+        .. versionadded:: 2.0
+        """
+        return datetime.datetime.now(datetime.timezone.utc) - self.start
```
Comment (veni-vidi-code) [inline nextcord/activity.py:720]: https://nextcord.readthedocs.io/en/latest/api.html#nextcord.utils.utcnow should be prefered

**Review by veni-vidi-code — CHANGES_REQUESTED** [review]

File: nextcord/activity.py (hunk 1 above)
Comment (ooliver1) [reply nextcord/activity.py:226]: Ah fuck

**Review by veni-vidi-code — APPROVED** [review]

**Comment by ooliver1** [comment]
@TAG-Epic

**Comment by veni-vidi-code** [comment]
LGTM

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
Maybe add elapsed to repr?

**Comment by ooliver1** [comment]
This is quite stale and it's honestly useless, only made it to attempt the issue

**Pull request closed without merging**